src/CloudNimble.DotNetDocs.Sdk/CloudNimble.DotNetDocs.Sdk.csproj

```diff
@@ -1,15 +1,17 @@
-﻿<Project Sdk="Microsoft.Build.NoTargets/3.7.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
         <!-- SDK packages don't compile code -->
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
You only actually used the net472 and net8.0 builds of your Tasks in your Targets code, so I slimmed things down to just those two TFMs. The code that is below will easily scale to more TFMs if you want to add them, though.